src/test/scala/uk/gov/nationalarchives/tdr/api/routes/RouteAuthenticationSpec.scala

```diff
@@ -39,4 +50,49 @@ class RouteAuthenticationSpec extends AnyFlatSpec with Matchers with ScalatestRo
       status shouldEqual StatusCodes.OK
     }
   }
+
+  "The db" should "return ok when there is at least one transferring body in the db" in {
+    addTransferringBody(UUID.randomUUID(), "MOCK Department", "Code")
+    Get("/healthcheck-full") ~> route ~> check {
+      status shouldEqual StatusCodes.OK
+    }
+  }
+
+  "The db" should "return 500 Internal Server Error if there are no transferring bodies in the db" in {
+    Get("/healthcheck-full") ~> route ~> check {
+      status shouldEqual StatusCodes.InternalServerError
+    }
+  }
+
+  "The db" should "return 500 Internal Server Error if the db is down" in {
+    object DbConnectionTest {
+      implicit val passwordCache: CaffeineCache[String] = CaffeineCache[String](CacheConfig())
+      val slickSession: SlickSession = SlickSession.forConfig("consignmentapi")
+
+      def db: JdbcBackend#DatabaseDef = {
+        val db = slickSession.db
+        db.source match {
+          case hikariDataSource: HikariCPJdbcDataSource =>
+            val configBean = hikariDataSource.ds.getHikariConfigMXBean
+            getPassword match {
+              case Failure(exception) => throw exception
+              case Success(password) =>
+                configBean.setPassword(password)
+                db
+            }
+          case _ =>
+            db
+        }
+      }
+    }
+
+    val testDb = DbConnectionTest.db
+    val route = new Routes(ConfigFactory.load(), testDb).route
```

Comment:
Because this parameter is not used in the `Routes` class, this is testing the same thing as the test above. I think checking that we get an error if the service throws an exception when the table is empty is enough and you can get rid of this one.